src/main/java/com/conveyal/datatools/manager/jobs/DeployJob.java

```diff
@@ -561,7 +575,14 @@ private void replaceEC2Servers() {
                     // If an error occurred while monitoring the initial server, fail this job and instruct user to inspect
                     // build logs.
                     status.fail("Error encountered while building graph. Inspect build logs.");
-                    ServerController.terminateInstances(ec2, graphBuildingInstances);
+                    try {
+                        ServerController.terminateInstances(ec2, graphBuildingInstances);
+                    } catch (AmazonEC2Exception e){
+                        status.message = String.format(
+                            "%s During job cleanup, the graph building instance was not properly terminated!",
+                            status.message
+                        );
+                    }
```

Comment:
Rather than surrounding each call to terminateInstances in a try/catch, we should just have the terminateInstances method itself contain the try/catch, check the `TerminateInstancesResult` and return a success boolean. I don't think the current return type is really used anywhere.